release-notes/major11/11.2.0/valtimo-backend-libraries.md

```diff
@@ -0,0 +1,76 @@
+# Backend libraries 11.2.0
+
+## New Features
+
+The following features were added:
+
+* **Transactional outbox**
+
+  Valtimo now supports an implementation of
+  the [transactional outbox pattern](https://microservices.io/patterns/data/transactional-outbox.html). See
+  [this page](/getting-started/modules/core/outbox.md) for more information.
+
+* **Added Audit CloudEvents**
+
+  Several CloudEvents have been added for auditing purposes. These events are:
+  - DocumentCreated
+  - DocumentUpdated
+  - DocumentViewed
+  - TaskCompleted
+
+* **Added Outbox message publisher for RabbitMQ**
+
+  A RabbitMQ (AMQP 0.9.1) message publisher has been in the `outbox-rabbitmq` module. More information can be found [here](/getting-started/modules/core/outbox-rabbitmq.md)  
+
+* **Added option for custom process variable name to download document plugin action in documentAPI**
```

Comment:
Can you add these release notes to branch: `development/11.2.0`?

Reply:
Changed it! Thanks for noticing